addons/im_livechat/populate/mail_channel.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models
+from odoo.tools import populate
+
+
+class Channel(models.Model):
+    _inherit = "mail.channel"
+    _populate_dependencies = ["im_livechat.channel"]
+
+    _populate_sizes = {
+        'small': 200,
+        'medium': 1.105e5,
+        'large': 1.51e6,
+    }
```

Comment:
When we don't have im_livechat we will have the populate size from the mail, and once im livechat is installed it will take the populate size from the livechat populate.